Update amp-geo doc with preset-us-ca support (#27928)
ampproject/amphtml: extensions/amp-geo/amp-geo.md

In addition to user specifed country groups `amp-geo` supports preset country lists. See [`amp-geo-presets.js`](./0.1/amp-geo-presets.js) for the available preset lists.

##### U.S. California Detection

> As a special case, `amp-geo` provides the [ISO 3166-2](https://en.wikipedia.org/wiki/ISO_3166-2) information when the request is determined to be from the U.S. California. This is supported via the `preset-us-ca` preset list.

Reviewer: Thanks for updating the documentation to include this! I've suggest a rephrasing to avoid passive voice below.  

```suggestion
The `amp-geo` component provides the [ISO 3166-2](https://en.wikipedia.org/wiki/ISO_3166-2) information when it determines a request is from California. The `preset-us-ca` value from the preset list supports this feature.
```
